deny.toml

```diff
@@ -48,6 +48,7 @@ skip = [
     { name = "ndk-glue", version = "0.5" },             # from winit v0.26.1
     { name = "ndk-sys", version = "0.2" },              # from winit v0.26.1
     { name = "stdweb", version = "0.1" },               # from rodio v0.15.0
+    { name = "wasi",  version = "0.11"},                # from winit v0.26.1
```

Comment:
it is better to ignore the older version, in this case the 0.10 from ahash/getrandom

Reply:
```suggestion
    { name = "wasi",  version = "0.10"},                # from ahash v0.7.6
```